lambdas/InspectMediaLambdaFunction/audio.js

```diff
@@ -0,0 +1,95 @@
+const ebur128 = require('./ebu-r-128');
+const ffprobe = require('./ffprobe');
+const mpck = require('./mpck');
+
+/** @typedef {import('./index').InspectTask} InspectTask */
+
+/**
+ * @typedef {object} AudioInspection
+ * @property {number} [Duration]
+ * @property {string} [Format]
+ * @property {number} [Bitrate]
+ * @property {number} [Frequency]
+ * @property {number} [Channels]
+ * @property {string} [Layout]
+ * @property {string} [Layer]
+ * @property {number} [Samples]
+ * @property {number} [Frames]
+ * @property {number} [LoudnessIntegrated]
+ * @property {number} [LoudnessTruePeak]
+ * @property {number} [LoudnessRange]
+ */
+
+module.exports = {
+  /**
+   * @param {InspectTask} task
+   * @param {string} filePath
+   * @returns {Promise<AudioInspection>}
+   */
+  inspect: async function inspect(task, filePath) {
+    /** @type {AudioInspection} */
+    const inspection = {};
+
+    // FFprobe data
+    try {
+      const probe = await ffprobe.inspect(filePath);
+      const stream = probe.streams.find((s) => s.codec_type === 'audio');
+
+      if (stream) {
+        Object.assign(inspection, {
```

Comment:
This is now coercing a few properties to numbers in the actual return value. Previously they were using whatever type was in the JSON returned by FFprobe. I haven't seen any cases where something like `bit_rate` is a value that can't correctly be coerced to a number, but FFmpeg does sometimes return ratios as a string (like `"24000/1001"`).

I can't find any sort of list of what that may happen. So there's some risk here that the coercion would fail, but I don't think any of our client apps would handle that case correctly right now anyway, so I don't believe this could cause any regressions.